neo/Consensus/ConsensusService.cs

```diff
@@ -20,21 +22,25 @@ public class Start { }
         public class SetViewNumber { public byte ViewNumber; }
         internal class Timer { public uint Height; public byte ViewNumber; }
 
+        private const byte ContextSerializationPrefix = 0xf4;
```

Comment:
It seems like this belongs in the `Prefixes` class in Neo.Persistence.LevelDB namespace.

Reply:
Let's keep this moving, in the future we polish this part.

Reply:
I implemented it in #575